30_regexp_lab/relab.txt

```diff
@@ -1,17 +1,18 @@
 Use grep to print only the lines from "input" for which:
 
 1. There is a binary number in the line
-
+grep -E [10] input.txt
 2. There is an even binary number in the line
-
+grep -E '0$' input.txt
 3. Line includes a hexadecimal number
-
+grep -E '0x' input.txt
```

Comment:
same as above it's gonna work for this file, but if you want something more generic you can use the same trick that we used for binary :)